app/controllers/settings_controller.rb

```diff
@@ -256,6 +256,25 @@ def twitter_disconnect
     return redirect_to "/settings"
   end
 
+  def keybase_auth
```

Comment:
`raise SomethingAppropriate unless Keybase.enabled?`

Reply:
what is SomethingAppropriate here? I'm not familiar with errors in ruby

Reply:
I totally guessed wrong about your familiarity, sorry about that. :) I was writing quickly and didn't want to stop to look it up. `ActionController::RoutingError.new("Keybase not enabled")`

(This is a belt-and-suspenders check that's redundant with disabling the routing, but I'm a little paranoid around security features like authorization.)